### DISCUSSION TITLE
[WAD] City Themed Wad

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#214] [Other Content Releases]
Greetings so If you were looking for a City themed WAD well you're in the right place!

I update the WAD with basically every Map I make, so updates happen somewhat regularly!

Note: Don't mind the name it's actually a WAD for basically a lot of things, but it's mainly for Citys!

# Changelog

> 1st March 2023
> 
> Added 11 new Textures/Varients
> Edit: I edited the Sand after burning my eyes from seeing it in TB

> 7th April 2023
> 
> Added 19 new Textures/Varients.
> Therefore putting the Total count of all the Textures to 82 Textures.

# [Download](https://github.com/nzp-team/nzportable/files/11179101/Ultimate.Wad.zip)

**Comment by Riuk1z** [comment]
and where does each one go?

**Reply by Paindeluxe** [reply]
Well, it‘s a WAD you can put it anywhere then just add it to textures in TrenchBroom. There‘s no real constraint on where to put it.

**Reply by Paindeluxe** [reply]
The Folder is the uncompiled Version of the WAD, useful for when you want to make additions to it, or remove it… you can just do anything to it, but it won’t reflect the changes in the .WAD until you repackage it as a .WAD
